server/src/models/Pokestop.js

```diff
@@ -135,11 +135,20 @@ class Pokestop extends Model {
         onlyEventStops,
         onlyConfirmed,
         onlyAreas = [],
+        onlyExcludeGrunts,
+        onlyExcludeLeaders,
       },
     } = args
     const midnight = getUserMidnight(args)
     const ts = Math.floor(Date.now() / 1000)
 
+    const rocketGruntIDs = [
+      4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23,
+      24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 47, 48,
+      49, 50,
+    ]
+    const rocketLeaderIDs = [41, 42, 43, 44]
```

Comment:
Let's access these IDs in a more dynamic way. They're available on the `Event` object declared at the top, it should be typed correctly for intellisense. It's probably "fine" if you want to do the filtering for IDs in every call to this method but you could also just cache them in the `Event` object.

Start here:
https://github.com/WatWowMap/ReactMap/blob/8be85b68cd6797cdfd8228627d50ab6bd0d613a1/server/src/services/EventManager.js#L322-L334

Then filter them here to get all IDs for both grunts and executive/giovanni and save them in some new properties. Then you can access the cache here.

Reply:
Alright, its now getting the ID's dynamically - I'm using `Event.rocketGruntIDs` & `Event.rocketLeaderIDs`. 
I'm not 100% sure if thats all you meant by caching the values, or if you want me to actually `getCache` somewhere.